indigo_api/management/commands/migrate_taxonomies.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management import BaseCommand
+from indigo_api.models import TaxonomyTopic, TaxonomyVocabulary, Work
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        for t in TaxonomyVocabulary.objects.all().iterator():
+            root = TaxonomyTopic.add_root(name=t.title)
+            for vocabulary in t.topics.all():
+                if vocabulary.level_1:
+                    child = TaxonomyTopic.add_child(root, name=vocabulary.level_1)
+                    if vocabulary.level_2:
+                        TaxonomyTopic.add_child(child, name=vocabulary.level_2)
```

Comment:
Isn't it safer and simpler to, at this point, use `for work in vocabulary.works.all()` to adjust the works to use the new `child` (either at level 1 or level 2).

Reply:
@longhotsummer Yeah. Forgot about the reverse relationship. Changed.